backend/audit/test_models.py

```diff
@@ -9,6 +9,6 @@
 class SingleAuditChecklistTests(TestCase):
 
     def test_str_is_id_and_ein(self):
```

Comment:
Minor: consider changing function name to "test_str_is_id_and_uei"

Reply:
Thanks @NicholasKramer, let's use :octocat: suggestion for things like this.

```suggestion
    def test_str_is_id_and_uei(self):
```